game-core/src/main/java/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -1191,6 +1182,17 @@ private void updateUnitsThatCanMoveOnRoute(final Collection<Unit> units, final R
     unitsThatCanMoveOnRoute = new ArrayList<>(lastResult.getUnitsWithDependents());
   }
 
+  private Collection<Unit> getPossibleTransportsToLoad(
+      final Collection<Unit> units, final Route route) {
+    // TODO kev check for already loaded airTransports
+    if (MoveValidator.isLoad(units, dependentUnits, route, getData(), getCurrentPlayer())) {
```

Comment:
nit, could go for a ternary expression and make this a bit more concise. It adds more to each line but the single return statement at front should make it conceptually easier to understand (the reader knows right away we are returning). 
eg:
```
return MoveValidator.isLoad(units, dependentUnits, route, getData(), getCurrentPlayer())
    ? route.getEnd().getUnitCollection().getMatches(
          Matches.unitIsTransport().and(Matches.alliedUnit(getCurrentPlayer(), getData()))
    : List.of();
```

Reply:
I find your example harder to read. Note that I did refactor this to a helper to allow the assignment in the caller to be done to a final variable. I find overall the helper + two returns is more readable than the other alternatives like your suggestion. But I'm curious what others think.

Reply:
IMO depends on expectations and what a person is used to. The multiple return points has higher cognitive complexity, the two are otherwise equivalent. It's not easy to read either way.